assets/js/modules/analytics/components/common/ProfileSelect.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable sitekit/camelcase-acronyms */
```

Comment:
I don't think we should have file-wide exceptions for this rule, let's annotate the individual instances.